apps/nextjs-website/src/app/[[...slug]]/page.tsx

```diff
@@ -178,6 +180,16 @@ const Page = async ({ params }: PageParams) => {
   }
 
   const sections = pageProps.sections;
+
+  const redirectSection = sections.find(
+    (section): section is RedirectSectionData =>
+      section.__component === 'sections.redirect',
+  );
+
+  if (redirectSection?.redirectURL?.trim()) {
+    return <RedirectSectionComponent {...redirectSection} />;
+  }
```

Comment:
```suggestion
  const redirectSection = sections.find(section => section.__component === 'sections.redirect');

  if (redirectSection) {
    return <RedirectSectionComponent {...(redirectSection as RedirectSectionData)} />;
  }
```
Only cast when we're sure the data is indeed RedirectSectionData + simplify existence check